exercises/hardware_access_control/README.md

```diff
@@ -157,3 +158,5 @@ Where to go from here...
     You could have a go at adding these to the `gpio_access` compartment.
 - The interactions with `ledTaken` global in the `gpio_access` compartment aren't thread safe.
     You could take a look at `cheriot-rtos/examples/06.producer-consumer/` to learn how to use a futex to make it thread safe.
+- There is a technical interest group for Sunburst and a technology access programme run by UKRI that lowRISC is helping to adjudicate.
+    If you are interested in either of these please reach out to [info@lowrisc.org](mailto:info@lowrisc.org).
```

Comment:
This may make sense in a different document in the future, but since we only have one exercise. Here works for now.